src/util/storageOptions.tsx

```diff
@@ -35,6 +35,18 @@ export const getStorageDriverOptions = (
   );
 };
 
+export const getSupportedStorageDrivers = (
+  settings?: LxdSettings,
+): Set<string> => {
+  if (!settings) {
+    return new Set();
+  }
+
```

Comment:
```suggestion
```
I think `getStorageDriverOptions` is happy to receive empty settings, so we don't need an early exit here. That would simplify the control flow, as there is only one way through the function.

Reply:
good catch thanks :+1: